docker-compose.yml

```diff
@@ -40,12 +40,37 @@ slave1:
     - MESOS_ISOLATION=com_mesosphere_mesos_CalicoIsolator
     - MESOS_HOOKS=com_mesosphere_mesos_CalicoHook
     - ETCD_AUTHORITY=etcd:4001
+  dns:
+   - 192.168.255.254
+
+slave2:
```

Comment:
We already have a `slave2` below, why do we want one more? Can't we simply merge it with the one below?